azurelinuxagent/common/cgroupconfigurator.py

```diff
@@ -78,16 +91,14 @@ class CGroupConfigurator(object):
     NOTE: with the exception of start_extension_command, none of the methods in this class
     raise exceptions (cgroup operations should not block extensions)
     """
-    class __Impl(object):
+    class _Impl(object):
```

Comment:
I need access to some of the non-public members in the unit tests so removed 1 underscore from the name to avoid name mangling.